cirq-aqt/cirq_aqt/aqt_device.py

```diff
@@ -11,18 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# pylint: disable=line-too-long
 """Current device parameters for the AQT/UIBK ion trap device
 
 The device is based on a linear calcium ion string with
 arbitrary connectivity. For more information see:
 
-https://quantumoptics.at/en/publications/journal-articles.html
+[https://quantumoptics.at/en/publications/journal-articles.html](https://quantumoptics.at/en/publications/journal-articles.html){:.external}
```

Comment:
Does it work to use angle bracket syntax to avoid repeating the url?
```suggestion
<https://quantumoptics.at/en/publications/journal-articles.html>{:.external}
```
This is [supported](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses) by markdown, but not sure if the docsite allows it.

Reply:
sadly this isn't supported.  Docs say devsite follows github style markdown.